vsphere/replace_vm.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python3
+import fileinput
+
+with fileinput.FileInput('terraform.tfstate', inplace=True) as file:
```

Comment:
Can you please add a comment explaining why you have this file and why you are doing what you are doing.